structure/0055-mappings-team.md

```diff
@@ -0,0 +1,70 @@
+# Mappings team restructure
+
+## Summary
+
+The mappings team was originally created in [RFC 18][RFC18], but has since
+evolved: a triage team was created for [Quilt Mappings][QM], and the need to
+split the mappings team into smaller subteams has appeared. This RFC describes

```

Comment:
This wording needs some work.

```
but our needs have changed since then - a 
[Quilt Mappings][QM] triage team was created, 
and it became necessary to split the mappings team 
into smaller subteams.
```